pages/linux/getconf.md

```diff
@@ -0,0 +1,24 @@
+# getconf
+
+> getconf - get configuration values from your Linux system.
```

Comment:
```suggestion
> getconf - get configuration values from your Linux system.
> More information: https://manned.org/getconf.1.
```

- This change is [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout).
- More information should be provided to allow user explore command in details when needed.

Reply:
Teminology such as *mandatory* sounds intimidating. Perhaps would could rephrase that?

Reply:
I don't know. But I always divide fixes into two categories: mandatory (required by style guide or specification) and option (all others).